x-pack/plugins/reporting/server/routes/jobs.ts

```diff
@@ -35,7 +35,13 @@ export function registerJobInfoRoutes(reporting: ReportingCore) {
   router.get(
     {
       path: `${MAIN_ENTRY}/list`,
-      validate: false,
+      validate: {
```

Comment:
Note from @restrry: "platform doesn’t provide request params / body / query unless validation schema is provided"

Reply:
a couple of remarks
- page & size should be `number`, you parse them below
- `userHandler` might break type inference logic, at least it doesn't implement the same interface as wrappers in docs https://github.com/elastic/kibana/blob/e57f4d2b25ac95183e876ea0c305138ab119ea26/src/core/MIGRATION_EXAMPLES.md#migrating-hapi-pre-handlers.
 I'd recommend either implement it as [RequestHandlerWrapper](https://github.com/elastic/kibana/blob/517c34a7ebf6ad6e02dab08c04155f7e0ab17956/src/core/server/http/router/router.ts#L373) or pass user via `RequestHandlerContext`.

Reply:
[Our HoH](https://github.com/elastic/kibana/blob/master/x-pack/plugins/reporting/server/routes/lib/authorized_user_pre_routing.ts#L24) has roughly the same type passthrough interface. Is there something obvious I'm missing?